modules/fluid_properties/doc/content/source/ics/RhoFromPressureTemperatureIC.md

```diff
@@ -0,0 +1,14 @@
+# RhoFromPressureTemperatureIC
+
+Initial condition to set density from given pressure and temperature using the
+`SinglePhaseFluidProperties` interface.
+
+!syntax description /ICs/RhoFromPressureTemperatureIC
```

Comment:
I'm thinking we should delete these lines if you provide a more detailed description, as you have here.